addons/mail/static/src/models/messaging_notification_handler.js

```diff
@@ -100,6 +105,7 @@ registerModel({
                         case 'mail.channel/last_interest_dt_changed':
                             return this._handleNotificationChannelLastInterestDateTimeChanged(message.payload);
                         case 'mail.channel/legacy_insert':
+                            this.env.services.bus_service.updateChannels();
```

Comment:
I haven't checked the base code yet, but it's strange that a global call `updateChannels` without param does the job. I guess it relies on mail channels being known through the server?

Reply:
Yes indeed, channels are added by overriding the bus controller `_subscribe` method. Triggering the `/subscribe` route updates the channel subscriptions on the server.